Documentation/openshiftdocs/modules/metricsserverconfig.adoc

```diff
@@ -18,6 +18,8 @@ Appears in: link:clustermonitoringconfiguration.adoc[ClusterMonitoringConfigurat
 [options="header"]
 |===
 | Property | Type | Description 
+|audit|*Audit|Defines the audit configuration used by the Metrics Server instance. Possible profile values are: `metadata`, `request`, `requestresponse`, and `none`. The default value is `metadata`.
```

Comment:
I'd just stuck with one profile that will help with debugging for simplicity.
And I suggest disabling the audit logs by default and only propose a flag to enable them.
We could even enable them when debugging is enabled (if we already have a flag for that).

Reply:
+1 on not enabling this by default. It would make sense that a user wouldn't like to have debug mode enabled all the time so we should offer some option, bonus point on matching with debugging is enabled.